lego/apps/banners/views.py

```diff
@@ -0,0 +1,50 @@
+from rest_framework import status, viewsets
+from rest_framework.decorators import action
+from rest_framework.permissions import AllowAny
+from rest_framework.response import Response
+
+from lego.apps.banners.models import Banners
+from lego.apps.banners.serializers import BannersSerializer
+from lego.apps.permissions.api.permissions import LegoPermissions
+
+
+class BannersViewSet(viewsets.ModelViewSet):
+    lookup_value_regex = r"\d+"
+    serializer_class = BannersSerializer
+    queryset = Banners.objects.all()
+    permission_classes = [LegoPermissions, AllowAny]
+
+    def get_queryset(self):
+        if self.action == "current_private":
+            return Banners.objects.filter(current_private=True)
+        if self.action == "current_public":
+            return Banners.objects.filter(current_public=True)
```

Comment:
Why do we do this here, instead of directly filtering in the action handlers? And does it make a difference using `Banners.objects.filter...` instead of `super().get_queryset().filter...`?